iac-extensions/kubernetes/src/test/resources/checks/CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod.yaml

```diff
@@ -1,33 +1,4 @@
-apiVersion: v1
```

Comment:
Why did you remove these tests?

Reply:
Good catch. Removed because I debugged the last case and forgot to restore it. Now it's resorted.